src/commands/from_bson.rs

```diff
@@ -60,6 +61,10 @@ fn convert_bson_value_to_nu_value(v: &Bson, tag: impl Into<Tag>) -> Tagged<Value
         // TODO: Add Int32 to nushell?
         Bson::I32(n) => Value::Primitive(Primitive::Int(*n as i64)).tagged(tag),
         Bson::I64(n) => Value::Primitive(Primitive::Int(*n as i64)).tagged(tag),
+        Bson::Decimal128(n) => {
+            let decimal = Decimal::from_str(&format!("{}", n)).unwrap();
+            Value::Primitive(Primitive::Decimal(decimal)).tagged(tag)
```

Comment:
actually, if you could use ```Decimal::from_scientific``` it would be more correct. But the bson library is just not working right.